YouTag/YYTTagCell.swift

```diff
@@ -79,24 +83,8 @@ class YYTTagCell: UICollectionViewCell {
 		textField.leadingAnchor.constraint(equalTo: self.contentView.leadingAnchor, constant: 2.5).isActive = true
 		textField.bottomAnchor.constraint(equalTo: self.contentView.bottomAnchor, constant: -2.5).isActive = true
 		textField.topAnchor.constraint(equalTo: self.contentView.topAnchor, constant: 6).isActive = true  // Because the font is shifted upward
-        textField.addTarget(self, action: #selector(YYTTagCell.textChanged), for: .editingChanged) // adding text change selector
 	}
     
-    // This function will fire whenever the user is typing out a new tag to add to a song
-    @objc func textChanged(txtField: UITextField) {
-        let tagList = LibraryManager.getAll(.tags)
-        for i in 0 ..< tagList.count {
-            print("Tag: \(tagList[i])")
-        }
-        if (txtField.text!.length == 0) {
-            suggestionTableView.isHidden = true
-        } else {
-            suggestionTableView.isHidden = false
```

Comment:
I think removing this will break the suggestion code as you still have to use filterStrings function each time the array changes